assembl/static2/js/app/components/debate/common/header.jsx

```diff
@@ -12,12 +12,19 @@ class Header extends React.Component {
     const { locale } = this.props.i18n;
     const isPhaseCompleted = getIfPhaseCompletedByIdentifier(debateData.timeline, identifier);
     const closedPhaseName = getPhaseName(debateData.timeline, identifier, locale).toLowerCase();
+    const titlesArray = title.split('#!');
+    let localizedTitle = '';
+    titlesArray.forEach((t) => {
+      if (t.split('$!')[1] === locale) {
```

Comment:
```if (locale in t.split('$!')[1] { ... }```. At least the hack would be more resilient to whitespace or other chars in between.